entity-service/src/main/kotlin/com/egm/stellio/entity/model/EntityTypeInfo.kt

```diff
@@ -0,0 +1,25 @@
+package com.egm.stellio.entity.model
+
+import java.net.URI
+
+data class EntityTypeInfo(
+    val id: URI,
+    val type: String = "EntityTypeInformation",
```

Comment:
not sure if it should be `EntityTypeInformation` or `EntityTypeInfo`

Reply:
In 5.2.26 it is `EntityTypeInfo`

Reply:
Forget about the first comment.
I think it's `EntityTypeInformation` (also mentionned in 4.5.12)

Reply:
yes, but the data type is called EntityTypeInfo and I think that the example in the Annex uses EntityTypeInfo.